### PR TITLE
support multi header

### PR DESCRIPTION
**Pull request by ipcjs**
Input tag can not enter multiple lines of text, so it replaced the textarea label.

### CODE REVIEW

**Comment by ziahamza** [comment]
well, header can be useful for new downloads, so I like this change 👍

File: js/ctrls/modal.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ angular
     open: function(cb) {
       var self = this;
       this.uris = "";
-      this.downloadSettingsCollapsed = true;
+      this.downloadSettingsCollapsed = false;
```
Comment (morozov) [inline js/ctrls/modal.js]: @ipcjs was it done on purpose or accidentally? Doesn't seem related to the issue. The idea of #298 was to have all settings collapsed by default to save vertical space, and now it's broken.

/cc @ziahamza

Comment (ziahamza) [reply js/ctrls/modal.js]: doesnt seem to be related to this issue, I will revert it back to `true` @morozov !
